.github/workflows/ci-github-actions-self-hosted.yaml

```diff
@@ -115,13 +115,13 @@ jobs:
       fail-fast: false
       matrix:
         jobname: [
-            Clang15-MPI-CUDA-AFQMC-Offload-Real-Mixed, # auxiliary field, offload
-            Clang15-MPI-CUDA-AFQMC-Offload-Real,
-            Clang15-MPI-CUDA-AFQMC-Offload-Complex-Mixed,
-            Clang15-MPI-CUDA-AFQMC-Offload-Complex,
-            Intel21-MPI-CUDA-AFQMC-Real-Mixed, # auxiliary field, requires MPI
-            Intel21-MPI-CUDA-AFQMC-Complex-Mixed,
-            Intel21-MPI-CUDA-AFQMC-Real,
+            TeslaV100-Clang16-MPI-CUDA-AFQMC-Offload-Real-Mixed, # auxiliary field, offload
+            TeslaV100-Clang16-MPI-CUDA-AFQMC-Offload-Real,
+            TeslaV100-Clang16-MPI-CUDA-AFQMC-Offload-Complex-Mixed,
+            TeslaV100-Clang16-MPI-CUDA-AFQMC-Offload-Complex,
+            TeslaV100-ICX23-MPI-CUDA-Real-Mixed,
```

Comment:
ICX23->OneAPI23.x
Please use the exact x.

Reply:
Perhaps "OneAPI23"? We'll presumably update to the latest minor version shortly after release. If the minor version .x is in the name this creates administrative work updating the merge requirements on GitHub.

Reply:
OneAPI23 comes with classic icc and OneAPI icx, I used icx as a way to distinguish. Moving forward just OneAPI should be enough. Thoughts?